src/reducers.js

```diff
@@ -11,21 +10,10 @@ import {
   storeName as reportStoreName,
 } from './report';
 
-const identityReducer = (state) => {
-  const newState = { ...state };
-  return newState;
-};
-
-const createRootReducer = history =>
-  combineReducers({
-    // The authentication state is added as initialState when
-    // creating the store in data/store.js.
-    authentication: identityReducer,
```

Comment:
Why is this being removed?

Reply:
Similar to the other case, `initialize()` takes care of authentication and config.